modules/osd-applications-config-custom-domains.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_content-type: PROCEDURE
 [id="osd-applications-config-custom-domains_{context}"]
-= Configuring custom domains for applications
+= Configure custom domains for applications
 
-Custom domains are specific wildcard domains that can be used with {product-title} applications. The top-level domains (TLDs) are owned by the customer that is operating the {product-title} cluster. The Custom Domains Operator sets up a new `ingresscontroller` with a custom certificate as a second day operation. The public DNS record for this `ingresscontroller` can then be used by an external DNS to create a wildcard CNAME record for use with a custom domain.
+The top-level domains (TLDs) are owned by the customer that is operating the {product-title} cluster. The Custom Domains Operator sets up a new `ingresscontroller` with a custom certificate as a second day operation. The public DNS record for this `ingresscontroller` can then be used by an external DNS to create a wildcard CNAME record for use with a custom domain.
```

Comment:
Should `ingresscontroller` be ingress controller here if this is a general reference?